eZ/Publish/Core/Persistence/Legacy/Tests/Content/Location/Gateway/DoctrineDatabaseTest.php

```diff
@@ -27,6 +29,24 @@ protected function getLocationGateway()
         );
     }
 
+    /**
+     * Returns a ready to test DoctrineDatabase gateway.
+     *
+     * @return \eZ\Publish\Core\Persistence\Legacy\Content\Gateway\DoctrineDatabase
+     */
+    protected function getContentGateway()
+    {
+        $databaseGateway = new \eZ\Publish\Core\Persistence\Legacy\Content\Gateway\DoctrineDatabase(
```

Comment:
Can we import `\eZ\Publish\Core\Persistence\Legacy\Content\Gateway\DoctrineDatabase` class with some alias e.g. ContentDoctrineDatabase  ?

Reply:
Actually, alias was already in place but not used in this case. Fixed in 52d314b77841292844c4d2796bbb12cba21556f2